public/js/app.Ctrl.js

```diff
@@ -0,0 +1,201 @@
+App.controller('App.Ctrl', function($scope, $timeout, AppService) {
+    const colorCode = {1:"green", 2: "red", 3: "yellow", 4: "blue"};
```

Comment:
I changed colorCode to COLOR_CODE and capitalized all other constant variables.  However I kept the code numbers for scaling purposes and in case I wanted to change the colors or their order in the future.